VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionCommentsFragment.java

```diff
@@ -111,6 +114,10 @@ public void onClick(View v) {
                         router.showCourseDiscussionAddComment(context, discussionResponse, discussionThread);
                     }
                 });
+
+        if (isNewCommentPosted) {
```

Comment:
This flag should be set as `false` after the first use, as `onViewCreated()` may be called multiple times.